usecases/config/config_handler.go

```diff
@@ -103,11 +103,11 @@ type RuntimeOverrides struct {
 
 // UsageConfig holds configuration for usage data collection and upload
 type UsageConfig struct {
```

Comment:
suggestion: `s/UsageConfig/BillingUsageConfig/g` would be better name I think?. just "usage" seems so generic. no?